Sources/AsyncHTTPClient/HTTPClient.swift

```diff
@@ -290,6 +290,24 @@ public class HTTPClient {
 
     /// `HTTPClient` configuration.
     public struct Configuration {
+        /// Timeout configuration
+        public struct Timeout {
+            /// Specifies connect timeout.
+            public var connect: TimeAmount?
+            /// Specifies read timeout.
+            public var read: TimeAmount?
+
+            /// Create timeout.
+            ///
+            /// - parameters:
+            ///     - connect: `connect` timeout.
+            ///     - read: `read` timeout.
+            public init(connect: TimeAmount? = nil, read: TimeAmount? = nil) {
+                self.connect = connect
+                self.read = read
+            }
+        }
```

Comment:
Could be a bit cleaner to either put the type into an extension or put it next to the other types defined here -- i.e. at the bottom of `Configuration` rather than on the top? When reading config I'm first interested in what fields it has after all 🤔

Reply:
Good idea, thank!
Does it look like what you were thinking of?